src/benchmark_mediation.py

```diff
@@ -935,6 +935,222 @@ def medDML(y, t, m, x, trim=0.05, order=1):
     return list(raw_res_R[0, :5]) + [ntrimmed]
 
 
+def med_dml(x, t, m, y, k=4, trim=0.05, normalized=True, alpha=10**-4):
```

Comment:
for consistency with other methods, I would prefer the same behavior, i.e. cross-validation if there is a penalty (regularization=True), or alpha=0 if there is no penalty (regularization=False)